feat: mssql enhancements (#279)
GoogleCloudPlatform/database-assessment: docs/user_guide/sqlserver/collection_scripts.md

> #### Perfmon Requirements

Reviewer: ```suggestion
#### Perfmon Requirements (Optional)
```

- If you have your own perfmon counters capturing the following statistics or run on a SQL Server Product such as Amazon RDS or Google CloudSQL for SQL Server, skip to step b, otherwise proceed to step a.

> - Perfmon Data is not required. This collection can be safely skipped.

Reviewer: ```suggestion
** The Perfmon data collection process is optional and can be safely skipped.  However, there will be no right sizing information in the assessment report.
```

- From a command prompt session in "Administrator Mode" on the server you would like to collect data on, execute the following command:

* ManageSqlServerPerfmonDatset.bat
  The following parameters can be specified:
  - -operation \*\* Required (create, stop, delete, collect)
  - -instanceType \*\* Required (default, managed)
  - -mssqlInstanceName \*\* Required if instanceType is "managed" (should be the instance name without the server name)

To execute the perfmon collection:

        For a default instance:
            ManageSqlServerPerfmonDatset.bat -operation create -instanceType default

        For a named instance:
            ManageSqlServerPerfmonDatset.bat -operation create -instanceType managed -mssqlInstanceName [instance name]

The script will create a permon data set that will collect the above metrics at a 1 minute interval for 8 days. The dataset will automatically stop after 8 days of collection. To get the most accurate statistics, it would be good to have this collection run over the busiest time for the server.
<br/>

> #### Perform Collection

Reviewer: ```suggestion
#### Perform Collection (Optional)
```
